mobile/src/components/LargeButton.tsx

```diff
@@ -3,9 +3,9 @@ import { Text, StyleSheet, Pressable } from 'react-native';
 import { Colors } from '../styles/Colors';
 import { TextStyles } from '../styles/TextStyles';
 
-type LargeButtonProps = { text: string, onPress: () => void };
+type LargeButtonProps = {text: string, onPress: () => void, underline?: boolean };
 
-export const LargeButton: React.FC<LargeButtonProps> = ({ text, onPress }) => {
+export const LargeButton: React.FC<LargeButtonProps> = ({ text, onPress, underline }) => {
```

Comment:
minor thing, you can replace `underline` with `underline = false` to give it a default value here.

Copied here for clarity since I know it's weird syntax:
```ts
export const LargeButton: React.FC<LargeButtonProps> = ({ text, onPress, underline = false }) => {
```